frontend/src/__mocks__/mockCatalogModel.ts

```diff
@@ -1,15 +1,14 @@
+// This file is a workaround for webpack module parse error and will remove it as a part of RHOAIENG-18959
```

Comment:
This comment isn't accurate, that applies to the other mocks in `frontend/src/concepts/modelCatalog/mockData` which will be removed by @YuliaKrimerman 's PR https://github.com/opendatahub-io/odh-dashboard/pull/3743 which makes them unnecessary. These mocks `frontend/src/__mocks__/*` are here to stay for tests.